server/app/com/xsn/explorer/data/anorm/parsers/TransactionParsers.scala

```diff
@@ -32,11 +32,12 @@ object TransactionParsers {
       TransactionWithValues(txid, blockhash, time, size, sent, received)
   }
 
-  val parseTransactionInput = (parseFromTxid ~ parseFromOutputIndex ~ parseIndex ~ parseValue ~ parseAddresses)
-    .map {
-      case fromTxid ~ fromOutputIndex ~ index ~ value ~ addresses =>
-        Transaction.Input(fromTxid, fromOutputIndex, index, value, addresses)
-    }
+  val parseTransactionInput =
+    (parseFromTxid ~ parseFromOutputIndex ~ parseIndex ~ parseValue ~ parseAddresses ~ parseHexScript)
```

Comment:
This won't work unless you update the queries to include the script on the selects, `parseHexScript` will fail, it isn't that simple to go this way, let me think a bit about it.